### PR TITLE
Structopt

### PR DESCRIPTION
**Pull request by sornas**
I also switched to anyhow. Behold: `??`!

### CODE REVIEW

**Review by default-username-852 — CHANGES_REQUESTED** [review]
Some variants of entering wrong commands prints `Error` not in red while some other error variants print `error` in red text. I would prefer it if every error was prefaced with the same variant, preferably the red variant if possible.

File: mumctl/src/main.rs [hunk 1]
```diff
@@ -41,519 +40,476 @@ impl log::Log for SimpleLogger {
...
             "audio.output_volume" => {
                 if let Ok(volume) = value.parse() {
-                    error_if_err!(send_command(Command::OutputVolumeSet(volume))?);
+                    send_command(MumCommand::OutputVolumeSet(volume))??;
                     config.audio.output_volume = Some(volume);
                 }
             }
             _ => {
-                error!("unknown config value {}", name);
+                return Err(CliError::ConfigKeyNotFound(key))?;
```
[the remark this answers is not in the document]
Comment (sornas) [reply mumctl/src/main.rs:283]: I believe not. We need the `?` to get the `Into` to do its thing.

**Review by default-username-852 — APPROVED** [review]
Looks good!

File: mumctl/src/main.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (default-username-852) [reply mumctl/src/main.rs:300]: I take your word for it.

**Pull request merged**